packages/cmrjs/cmr-utils.js

```diff
@@ -120,6 +104,30 @@ async function parseXmlString(xml) {
   return (promisify(xml2js.parseString))(xml, xmlParseOptions);
 }
 
+const isECHO10File = (filename) => filename.endsWith('cmr.xml');
+const isUMMGFile = (filename) => filename.endsWith('cmr.json');
+
+/**
+ * Returns True if this object can be determined to be a cmrMetadata object.
+ *
+ * @param {Object} fileobject
+ * @returns {boolean} true if object references cmr metadata.
+ */
+function isCMRFile(fileobject) {
+  const cmrfilename = fileobject.name || fileobject.filename || '';
```

Comment:
Is fileobject.filename always available?

Reply:
I'm not 100% sure, the function is used as a filter in `getCmrFileObjs` which is used in `reconcileCMRMetadata` part of `granules/move` where the fileobjects are coming from the Granule model originally.  Can you think of a time that a cmr file in there would exist without a name or filename ?